### PR TITLE
Special case version

### PR DESCRIPTION
**Pull request by olingern**
- ~Fixes issue @arcanis and I talked about [here](https://github.com/yarnpkg/berry/issues/172#issuecomment-493013348)~
- Adds small bit about running the CLI ( it wasn't immediately intuitive to me ) after building

### CODE REVIEW

**Comment by arcanis** [comment]
Ahah #173 😄

File: README.md [hunk 1]
```diff
@@ -90,4 +90,10 @@ Clone this repository, then run the following commands:
 $> yarn build:cli
 ```
 
+Run your newly built CLI:
+
+```
+$> node ./packages/berry-cli/bin/berry.js
```
Comment (arcanis) [inline README.md]: Note that this shouldn't be needed. If you check the `.yarnrc` you'll see that we have a `yarn-path` settings that points towards `./scripts/run-yarn.js`, which in turns call either the local build if available, or the checked-in one if not. So just calling the global Yarn should work!

Comment (olingern) [reply README.md]: Oh, I didn't know that ... nice! I went looking around for a `/bin` like the previous version and turned up this. I can amend this to only include a bit about how the `yarn-path` is set and let you handle the allowance in #173

Comment (arcanis) [reply README.md]: Sounds good 👍

Comment (olingern) [reply README.md]: Updated :slightly_smiling_face:

**Pull request merged**